src/core/connection.c

```diff
@@ -1892,6 +1892,225 @@ QuicConnRestart(
     QuicCryptoReset(&Connection->Crypto, CompleteReset);
 }
 
+_IRQL_requires_max_(PASSIVE_LEVEL)
+QUIC_STATUS
+QuicConnSendResumptionTicket(
+    _In_ QUIC_CONNECTION* Connection,
+    _In_ uint16_t AppDataLength,
+    _In_reads_bytes_opt_(AppDataLength)
+        const uint8_t* AppResumptionData
+    )
+{
+    QUIC_STATUS Status;
+    uint32_t EncodedTransportParametersLength = 0;
+    uint8_t* TicketBuffer = NULL;
+    uint16_t AlpnLength = *(Connection->Crypto.TlsState.NegotiatedAlpn);
+
+    QUIC_TRANSPORT_PARAMETERS LocalTPCopy;
+    QuicCopyMemory(&LocalTPCopy, &Connection->LocalTP, sizeof(LocalTPCopy));
```

Comment:
I believe you should be able to simply change this to:
```suggestion
    QUIC_TRANSPORT_PARAMETERS LocalTPCopy = Connection->LocalTP;
```